docs/rst/installation/linux_installation.rst

```diff
@@ -2,3 +2,337 @@
+    To be able to run this application, make sure that :code:`pip3` binary installation directory is in your :code:`$PATH` (:code:`$HOME/.local/bin` is normally introduced while login on an interactive non-root shell).
+
+Dependencies
+------------
+
+This section deals with the required dependencies for each package included in the tool suite.
+
+.. important::
+
+    The *Library* package is required for building the remain tool packages.
+
+Library tool
+^^^^^^^^^^^^
+
+The *eProsima Fast DDS QoS Profiles Manager Library* requires Xerces.
+The package test suite depends on GTest.
+
+Xerces
+""""""
+
+`Xerces C++ <https://xerces.apache.org/xerces-c/>`_ is an Apache project that provides XML parsers and related components solutions for development.
+In Ubuntu, it can be installed running:
+
+.. code-block:: bash
+
+    sudo apt install libxerces-c-dev
+
+GTest (optional)
+""""""""""""""""
+
+GTest is a unit testing library for C++.
+By default, *eProsima Fast DDS QoS Profiles Manager* does not compile tests.
+It is possible to activate them with the opportune CMake configuration options when calling colcon or CMake.
+
+GTest installation can be performed either following the `GTest installation instructions <https://google.github.io/googletest/>`_, or including the `Gtest repository <https://github.com/google/googletest>`_ in the workspace directory, running:
+
+.. code-block:: bash
+
+    git clone https://github.com/google/googletest ~/fastdds_qosprofman_ws/src/googletest-distribution
+
+.. note::
+
+    If the colcon deployment is followed, and GTest has been installed in the workspace directory, the GTest dependency needs to be included in the *Library* ``colcon.pkg`` file.
+    Please check the :ref:`colcon build procedure <installation_build_source_linux>` for further information.
+
+CLI tool
+^^^^^^^^
+
+The *eProsima Fast DDS QoS Profiles Manager CLI* depends on the tool suite *Library* and also on the following package.
+
+Docopt
+""""""
+
+`docopt <http://docopt.org/>`_ is a CLI description language that defines the interface of the command line applications and generates a parser for it automatically.
+In Ubuntu, the dependency can be installed running:
+
+.. code-block:: bash
+
+    sudo apt install libdocopt-dev
+
+GUI tool
+^^^^^^^^
+
+.. warning::
+
+    This tool is still in development process.
+
+Documentation tool
+^^^^^^^^^^^^^^^^^^
+
+The *eProsima Fast DDS QoS Profiles Manager Documentation* depends on the tool suite *Library* and also on the following package.
+
+Doxygen
+"""""""
+
+`Doxygen <https://www.doxygen.nl/>`_ is a C++ documentation generator.
+It is required to build the :ref:`Library API reference <fastdds_qos_profiles_manager_lib_api_reference>`.
+In Ubuntu, the dependency can be installed running:
+
+.. code-block:: bash
+
+    sudo apt install doxygen
+
+Python3 virtual environment (recomended)
```

Comment:
I do not think this is the correct title. These dependencies are required. What is it optional is either installed them in a virtual environment or in the host directly. I suggest renaming it to `Python3 package dependencies`.

The `python3-venv` is recommended. Maybe this section can be kept but it needs to be split from the other mandatory dependencies.